Demo/Demo/AppDelegate.swift

```diff
@@ -5,6 +5,8 @@ import UIKit
 import PlayerControls
 
 typealias Props = ContentControlsViewController.Props
+fileprivate var progress: ContentControlsViewController.Props.Progress = 0
+
 func props() -> Props {
```

Comment:
Why not just pass progress to the function?

Reply:
@AndriiMoskvin fixed